kafka/consumer.go

```diff
@@ -150,6 +151,30 @@ func (cfg *ConsumerConfig) finalize() error {
 	if cfg.FetchMinBytes < 0 {
 		errs = append(errs, errors.New("kafka: fetch min bytes cannot be negative"))
 	}
+	if cfg.BrokerMaxReadBytes < 0 {
+		errs = append(errs, errors.New("kafka: broker max read bytes cannot be negative"))
+	}
+	if cfg.MaxPollPartitionBytes > 0 {
```

Comment:
nit: the first nested if check `cfg.MaxPollPartitionBytes > 0` is redundant, just `cfg.MaxPollPartitionBytes > 1<<30` is enough